content/geophysical_surveys/airborne_fdem/index.rst

```diff
@@ -3,14 +3,12 @@
 Airborne FDEM
 =============
 
-.. todo::
-
-    PLACEHOLDER: figure of an airborne system.
-
-.. topic:: Purpose
-
-    TODO
-
+.. figure:: ./images/afem_cover.jpg
+   :align: center
+   :scale: 40%
+   :name: afem_cover
+   
+.. Source: Jean M. Legault, 2015, Airborne Electromagnetic Systems – State of the Art and Future Directions, CSEG RECORDER, VOL. 40, No. 06.
```

Comment:
There isn't a `source` environment - the best approach is probably to add this to the bib